GP/toy_experiments/toy_sin_noise.py

```diff
@@ -1,54 +1,50 @@
 # Copyright Lee Group 2019
-# Author: Ryan-Rhys Griffiths
+# Author: Ryan-Rhys Griffiths (Modified by Alex Aldrick)
 """
-This module contains the code for benchmarking heteroscedastic Bayesian Optimisation on a number of toy functions.
+This module contains the code for benchmarking heteroscedastic Bayesian Optimisation on a 1D toy problem.
 """
 
 import matplotlib.pyplot as plt
 from matplotlib.ticker import MaxNLocator
 import numpy as np
+from numpy.core._multiarray_umath import ndarray
```

Comment:
Interesting, what's this?

Reply:
no idea, must have been while debugging, runs fine without it, so will remove